gitops-v2/new/push_aws.yaml

```diff
@@ -0,0 +1,29 @@
+parameters:
+  - name: awsRegion
+    type: string
+    default: ""
+  - name: awsServiceConnection
+    type: string
+    default: ""
+
+steps:
+  - task: AmazonWebServices.aws-vsts-tools.AWSShellScript.AWSShellScript@1
+    displayName: "Get ECR login info"
+    inputs:
+      workingDirectory: "."
+      awsCredentials: ${{ parameters.awsServiceConnection }}
+      regionName: ${{ parameters.awsRegion }}
+      scriptType: inline
+      inlineScript: |
+        set -e
+        SHA_SHORT=$(shaShort)
+
+        aws ecr describe-repositories --repository-names ${SERVICE_NAME} || aws ecr create-repository --repository-name ${SERVICE_NAME} --image-scanning-configuration scanOnPush=true
+        ECR_URI=$(aws ecr describe-repositories --repository-names ${SERVICE_NAME} | jq -r '.repositories[0].repositoryUri')
+        ECR_PASSWORD=$(aws ecr get-login-password --region ${AWS_REGION})
+        docker login -u AWS -p $ECR_PASSWORD $ECR_URI
```

Comment:
I think you can `aws ecr get-login-password --region ${AWS_REGION} | docker login -u AWS --password-stdin` so as not to expose the password.

Reply:
This should be handled in separate PR, only moving AWS to make it possible to run without the extension.

Reply:
Fixed.